package_def/add.py

```diff
@@ -0,0 +1,15 @@
+from package_data.data import inp_directories, inp_documents
```

Comment:
It was a good idea to change functions to a class. Please, restore a class: it will decrease the number of files from 4 to 1 and will not spread logic on files